sidecars/asciidoc/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM asciidoctor/docker-asciidoctor
```

Comment:
I think it should contain license as well

Reply:
@svor done :+1: